listenbrainz/webserver/views/user.py

```diff
@@ -194,6 +196,54 @@ def reports(user_name: str):
         user=user
     )
 
+@user_bp.route("/<user_name>/playlists")
+def playlists(user_name: str):
+    """ Show user playlists """
+    
+    if not current_app.config.get("FEATURE_PLAYLIST", False):
+        raise NotFound()
```

Comment:
Is this a kosher way of doing it, or is there a way to selectively enabled certain routes when we do register_blueprint in init.py ?(https://github.com/metabrainz/listenbrainz-server/pull/1199/files#diff-227596bda6895e00021eead6f7a61331225f684e197d853a287fa163c5673a8cR254)

Reply:
Seems fine to me.

Reply:
if we find that we need to do this in more places, we can have a `@requires_feature("FEATURE_PLAYLIST")` decorator that does the same thing